components/Map/CurrentOutivityMarker.js

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from "react";
```

Comment:
useState braucht ihr hier dann nicht mehr
```suggestion
import { useEffect } from "react";
```